src/services/io/podio/JEventProcessorPODIO.cc

```diff
@@ -94,7 +94,7 @@ JEventProcessorPODIO::JEventProcessorPODIO() {
             "B0TrackerRecHits",
 
             //
-            "ForwardRomanPotParticles",
+            "ForwardRomanPotRecParticle",
```

Comment:
```suggestion
            "ForwardRomanPotRecParticles",
```
We've tried to be consistent in use of plural for collections (even across edm4hep ecosystems).

Reply:
Okay, I made that change in both the factory, and of course in the PODIO code.

Reply:
Yeah, DRY and ODR are definitely violated.